clang/lib/Analysis/ThreadSafety.cpp

```diff
@@ -887,7 +887,7 @@ class LockableFactEntry : public FactEntry {
   }
 };
 
-class ScopedLockableFactEntry : public FactEntry {
+class ScopedCapabilityFactEntry : public FactEntry {
```

Comment:
The other derived class is still called `LockableFactEntry`. We should either leave this or change both.